widget/playground/src/store/config.ts

```diff
@@ -98,24 +90,18 @@ export const initialConfig: WidgetConfig = {
     singleTheme: undefined,
     colors: {
       dark: {
-        background: undefined,
-        primary: undefined,
-        foreground: undefined,
-        success: undefined,
-        error: undefined,
-        warning: undefined,
-        surface: undefined,
-        neutral: undefined,
+        primary: '#1C3CF1',
+        secondary: '#2284ED',
+        neutral: '#222222',
+        background: '#010101',
+        foreground: '#FDFDFD',
       },
       light: {
-        background: undefined,
-        primary: undefined,
-        foreground: undefined,
-        success: undefined,
-        error: undefined,
-        warning: undefined,
-        surface: undefined,
-        neutral: undefined,
+        primary: '#1C3CF1',
+        secondary: '#469BF5',
+        neutral: '#E6E6E6',
+        background: '#FDFDFD',
+        foreground: '#010101',
```

Comment:
Why do we need the changes here? Shouldn't it work with `undefined`? 
(Also if needed, could you read from one of the presets?)

Reply:
fixed